### PR TITLE
Add developer payload support

### PR DESCRIPTION
**Pull request by rostopira**
Implementation of FR #70
Attempt no 2

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -34,7 +34,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.billingclient:billing:1.1'
```
Comment (hyochan) [inline android/build.gradle]: It seems you've downgraded `billing` API. Not using above and importing `AIDL` file seperately is deprecated from android iap sdk from v3.

Comment (hyochan) [reply android/build.gradle]: Actually, I think we had some miss communication in #73. I think the gradle `dep` should be used without separate importing.

**Comment by hyochan** [comment]
@rostopira Could you kindly see additional request? I'm sorry about the miss communication I've made previously.

**Comment by rostopira** [comment]
I don't have time for it currently, I will provide another PR some time later

**Pull request closed without merging**